frontend/src/entry/model/Files.tsx

```diff
@@ -48,8 +95,149 @@ export default function Files({ model }: FilesProps) {
   return (
     <>
       <Container sx={{ my: 2 }}>
-        <Stack direction={{ xs: 'column' }} spacing={2} justifyContent='center' alignItems='center'>
-          {entryFilesList}
+        <Stack spacing={2}>
+          <Box display='flex'>
+            <Box ml='auto'>
+              <Stack direction={'row'}>
+                <Button
+                  variant='text'
+                  onClick={handleMenuButtonClick}
+                  endIcon={anchorEl ? <ExpandLess /> : <ExpandMore />}
+                >
+                  <Stack sx={{ minWidth: '150px' }} direction={'row'} spacing={2} justifyContent={'space-evenly'}>
+                    {ASCorDESC === 'ASC' ? (
```

Comment:
After doing the type change for `SortingDirection` then you should be able to do:

`ascOrDesc === SortingDirection.ASC`